src/components/StepVisual.tsx

```diff
@@ -13,14 +13,15 @@ export const StepVisual: FC<StepVisualProps> = ({
   bgColorClass = "bg-grey-500",
 }) => {
   const SHAPE_CLASSES =
-    shape === "square" ? "clip-standing-square" : "clip-circle-50 -scale-[87%]";
+    shape === "square" ? "clip-standing-square" : "rounded-full -scale-[87%]";
```

Comment:
I had to change that to `rounded-full` because it did hide the element completely in Firefox.

Reply:
Interesting, I didn't have that issue in Firefox. But good choice, there is no reason anymore to use the custom `clip-circle-50` while the `rounded-full` creates the same result. It was in there because I had first considered a different approach where it would have made sense.